File: 1550-three-consecutive-odds/1550-three-consecutive-odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int combo = 0;
        for (int num : arr){
            if (num % 2)
                ++combo;
            else combo = 0;
            
            if (combo == 3) return true;
        }
        
        return false;
    }
};

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        count = 0
        for num in arr:
            if num % 2:
                count += 1
            else:
                count = 0
            
            if count == 3:
                return True

        return False